src/VisualStudio/CSharp/Impl/Snippets/SnippetExpansionClient.cs

```diff
@@ -81,6 +84,12 @@ public override int GetExpansionFunction(IXMLDOMNode xmlFunctionNode, string bst
                 case "GenerateSwitchCases":
                     pFunc = new SnippetFunctionGenerateSwitchCases(this, SubjectBuffer, bstrFieldName, param);
                     return VSConstants.S_OK;
+                case "ArgumentValue":
```

Comment:
the naming (so far) isn't too clear to me.  but maybe this will make more sense later on.

Reply:
❔ Any requested changes here?

Reply:
➡️ Moved under the umbrella of #50760